### DISCUSSION TITLE
Cura sliced models print twice as fast

### DISCUSSION THREAD

**Discussion by naasking** [supermerill/SuperSlicer#1701] [Q&A]
Just trying out SuperSlicer. Setup was straightforward enough, I input filament parameters, machine limits, etc. for my Marlin-based printer. Then tried to slice a calibration cube and noticed that estimated print time was 44 mins. That seemed long, so I compared against Cura that I've been running for awhile, and that estimated just under 20 mins for the same model and parameters.

Actual print time for the SS sliced model was bang on at 44 mins, and the Cura-sliced model was about 21 mins. I'm not sure where this print speed discrepancy is coming from.

Even if I crank the SS print and move speed settings for perimeters and infill to near machine limits, I can't get below 40 mins print time estimate. I must be missing something obvious, so does anyone have any ideas?

**Comment by supermerill** [comment]
You forgot to post your project (.3mf) file (save as).

**Comment by naasking** [comment]
Sorry about that, it was just the 20mm standard calibration cube. Github won't let me attach a .3mf file so I zipped it. Print speed settings basically match what I see in Cura. Thanks for any tips you can give! I know SS provides a lot more fine-grained control over slicing so there must be some options I just never had in Cura that are tripping me up here.

[Calibration cube.zip](https://github.com/supermerill/SuperSlicer/files/7412900/Calibration.cube.zip)

**Reply by supermerill** [reply]
It's slow because of the filament "short layer time" (under filament -> cooling)
These are here to let some time to the filament to cool before printing again on it.

If you want to print really fast disregarding that, set the time goal to 0.

For more comparison, I need also the cura project.

**Reply by naasking** [reply]
Thanks, that got it down to 28mins after tweaking a few more advanced settings. Cura project attached.

[CFFFP_xyzCalibration_cube.zip](https://github.com/supermerill/SuperSlicer/files/7414675/CFFFP_xyzCalibration_cube.zip)

**Reply by supermerill** [reply]
I see that the difference comes mainly from the infill.
I don't know how cura manage to calculate 1min for all the infill, but it seems very low, even more with a 60mm/s speed and with many big retract/unretract

I think cura also don't usa any wipe nor lift for the retraction parameters.

**Reply by naasking** [reply]
The infill did print very fast on the Cura sliced model vs. SS sliced model when I was watching it print. I actually don't think the Cura model did a proper gyro infill though, as I think I remember it being very messy and not matching the SS infill. I don't normally use gyro, I just used it this time to try and equalize the parameters across the two programs, so I'm not 100% sure what to look for. Cura estimated print time was always around 20 mins regardless of infill pattern though.

If I remove wiping and Z lift, ETA drops to 24m in SS which is pretty close to Cura. Cura's actual print time is always a bit underestimated, so the extra 2 mins this should probably be added to infill. I can provide a short video of each printing if that might help clarify what's actually happening.

**Reply by supermerill** [reply]
> I can provide a short video of each printing if that might help clarify what's actually happening.

Don't bother too much. I believe you.
